content/applications/hr/recruitment/recruitment-flow.rst

```diff
@@ -0,0 +1,189 @@
+================
+Recruitment flow
+================
+
+When a prospective employee applies for a job in Odoo, there is a pre-configured process from the
+:ref:`initial inquiry <recruitment/new>` to :ref:`creating a new employee
+<recruitment/new-employee>` once hired. The following outlines the default recruitment *flow*, or
+sequence of events, for Odoo's *Recruitment* application.
+
+.. important::
+   The following is based on Odoo's default recruitment pipeline. Be advised that if
+   :ref:`modifications are made <recruitment/customize-stages>` to the pipeline, the following will
+   differ.
+
+.. _recruitment/new:
+
+New
+===
+
+All applicants appear in the :guilabel:`New` stage, whether submitted online or if the applicant is
+:doc:`manually entered by a recruiter <add-new-applicants>`.
+
+When the applicant's card is created, Odoo automatically populates the
+:guilabel:`Subject/Application`, the applicant's :guilabel:`Name`, :guilabel:`Email`,
+:guilabel:`Mobile` number, and :guilabel:`LinkedIn Profile` on the applicant's card. This
```

Comment:
OMG you are right! You have to add EITHER a LinkedIN OR a Resume!!! I think this must have been changed because I swear, every time I was testing in my own blank DB's for my tour I always had to enter a fake LinkedIn profile. This is why another set of eyes is always helpful!